scipy/stats/_multivariate.py

```diff
@@ -507,6 +512,110 @@ def pdf(self, x, mean=None, cov=1, allow_singular=False):
         out = np.exp(self._logpdf(x, mean, psd.U, psd.log_pdet, psd.rank))
         return _squeeze_output(out)
 
+    def _cdf(self, x, mean, cov, maxpts, abseps, releps):
+        """
+        Parameters
+        ----------
+        x : ndarray
+            Points at which to evaluate the cumulative distribution function.
+        mean : ndarray
+            Mean of the distribution
+        cov : array_like
+            Covariance matrix of the distribution
+        maxpts: integer
+            The maximum number of points to use for integration
+        abseps: float
+            Absolute error tolerance
+        releps: float
+            Relative error tolerance
+
+        Notes
+        -----
+        As this function does no argument checking, it should not be
+        called directly; use 'cdf' instead.
+
+        .. versionadded:: 0.19.0
+
+        """
+        lower = np.full(mean.shape, -np.inf)
+        # mvnun expects 1-d arguments, so process points sequentially
+        func1d = lambda x_slice: mvn.mvnun(lower, x_slice, mean, cov,
+                                           maxpts, abseps, releps)[0]
+        out = np.apply_along_axis(func1d, -1, x)
+        return _squeeze_output(out)
+
+    def logcdf(self, x, mean=None, cov=1, allow_singular=False, maxpts=100000,
+               abseps=1e-6, releps=1e-6):
```

Comment:
Originally, I found that `maxpts=1000*d` is insufficient to even pass the unit tests and therefore I increased the default value. Previously, I dropped the `*d` because the default parameter handling looks a bit uglier, but I agree that it is much better to scale the default for `maxpts`.
So now I ran a small [simulation](https://gist.github.com/asnelt/1440ce76afe0f7e583c0ddbf00b6a91b) to see how the maximum estimated error depends on `maxpts` and the dimension `dim` of the distribution when sampling the covariance matrix from an inverse Wishart distribution. I used `mvn.mvndst` instead of `mvn.mvnun` because the latter does not expose an error estimate whereas the former returns a 99% confidence level error estimate. It looks like the default `abseps=1e-6` requires at least `maxpts=100000*dim`. So now I use this as a default.